src/components/Property/RaiseWorkOrder/RaiseWorkOrderForm.test.js

```diff
@@ -102,4 +103,31 @@ describe('RaiseWorkOrderForm component', () => {
 
     expect(asFragment()).toMatchSnapshot()
   })
+
+  it('should render without possibility to choose budget code', async () => {
```

Comment:
Nitpick but could we add info on what controls this to the description, e.g. `...for a user without the budget code officer permission`.